### PR TITLE
Add Python packages krtc, psdm_qs_cli

### PR DESCRIPTION
**Pull request by klauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

cc @hhslepicka

### CODE REVIEW

File: recipes/krtc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% set name = "krtc" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
```
Comment (hhslepicka) [inline recipes/krtc/meta.yaml]: we should remove the `v` from `v{{ version }}`

File: recipes/psdm_qs_cli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{% set name = "psdm_qs_cli" %}
+{% set version = "0.2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: v{{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (hhslepicka) [inline recipes/psdm_qs_cli/meta.yaml]: Same as above, if we switch the package upstream to use the prefix at the tag we also need to change it here.

**Comment by hhslepicka** [comment]
I agree to be a maintainer of this package.

File: recipes/krtc/meta.yaml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 {% set name = "krtc" %}
-{% set version = "0.1.3" %}
+{% set version = "0.1.4" %}
 
 package:
   name: {{ name|lower }}
-  version: {{ version }}
+  version: v{{ version }}
```
Comment (hhslepicka) [inline recipes/krtc/meta.yaml]: This `v` should not be here

File: recipes/psdm_qs_cli/meta.yaml [hunk 4]
```diff
@@ -1,13 +1,13 @@
 {% set name = "psdm_qs_cli" %}
-{% set version = "0.2.9" %}
+{% set version = "0.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: v{{ version }}
```
Comment (hhslepicka) [inline recipes/psdm_qs_cli/meta.yaml]: we should also remove the `v` from here

**Comment by hhslepicka** [comment]
@conda-forge/help-python this is ready for a review

**Comment by klauer** [comment]
Noting that pykerberos is not available on windows through conda-forge, I'll make `krtc` not build for it and `psdm_qs_cli` include `krtc` on all platforms except for windows.

File: recipes/krtc/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: b715b07881316842c359d2377222c476f5e24d77b194b7eeaeb6e741a1204d95
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
```
Comment (scopatz) [inline recipes/krtc/meta.yaml]: This can be a noarch python package

File: recipes/psdm_qs_cli/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: be225c77d5a0941c24bb48e127ee15a9f8a880e134206b38b37ede996c91c7d2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (scopatz) [inline recipes/psdm_qs_cli/meta.yaml]: This can also be `noarch: python`

File: recipes/krtc/meta.yaml (hunk 5 above)
Comment (hhslepicka) [reply recipes/krtc/meta.yaml]: @scopatz it can't as `pykerberos` is not available for Windows.

Comment (klauer) [reply recipes/krtc/meta.yaml]: Sadly, krtc won't build for windows. From my comment above:

> Noting that pykerberos is not available on windows through conda-forge, I'll make `krtc` not build for it and `psdm_qs_cli` include `krtc` on all platforms except for windows.

Do you have a better solution?

Comment (dopplershift) [reply recipes/krtc/meta.yaml]: It was my understanding that you could leave *this* package as noarch and just let the lack of dependency keep it from installing on windows.

Comment (hhslepicka) [reply recipes/krtc/meta.yaml]: @dopplershift but that will create an error with the CI here at the recipe. Would that just be ignored? Seems quite wrong to show a package as available and it fails to install as dependencies are not available. Can we move forward with it not being `noarch`?

Comment (npavlovikj) [reply recipes/krtc/meta.yaml]: `noarch` recipes are built only on Linux, so the failures on Windows and OSX can be ignored during the build.

Comment (klauer) [reply recipes/krtc/meta.yaml]: Thanks so much for clarifying, @npavlovikj. I'll do that.

**Pull request merged**